pkg/ecdsa/tss/protocol_announce.go

```diff
@@ -44,9 +48,25 @@ func AnnounceProtocol(
 			case msg := <-announceInChan:
 				// Since broadcast channel has an address filter, we can
 				// assume each message come from a valid group member.
-				receivedMemberIDs[msg.SenderID.String()] = msg.SenderID
+				publicKey, err := msg.SenderID.PublicKey()
+				if err != nil {
+					logger.Errorf(
+						"could not get public key for member [%s] of keep [%v]: [%v]",
+						msg.SenderID.String(),
```

Comment:
Shouldn't formatting [%s] be enough to convert bytes to string? Do we need additional .String()?

Reply:
Yes, from Go docs (https://golang.org/pkg/fmt/):
> 5. If an operand implements method String() string, that method will be invoked to convert the object to a string, which will then be formatted as required by the verb (if any). 

I've left `.String()` to keep it consistent with the rest of the code in `keep-ecdsa` but I am also fine with dropping it.

Reply:
I'd vote for dropping it since it will be converted automatically.